flybywiresim/a32nx
# refactor(xml): small xml refactoring

## Summary of Changes

* Move exterior and generic xml files to `src`, fix some XML issues in exterior xml
* Change brake fan button to generic template

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

* Make sure the brake fan button works
* Make sure exterior animations work


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
